SPECS/rubygem-multipart-post/rubygem-multipart-post.spec

```diff
@@ -30,10 +30,13 @@ gem install -V --local --force --install-dir %{buildroot}/%{gemdir} %{gem_name}-
 
 %files
 %defattr(-,root,root,-)
-%license %{gemdir}/gems/%{gem_name}-%{version}/LICENSE
+%license license.md
 %{gemdir}
 
 %changelog
+* Sat Mar 23 2024 Riken Maharjan <rmaharjan@microsoft.com> - 2.3.0-1
+- Upgrade to 2.3.0 - none
```

Comment:
what does "none" mean here?

Reply:
removed. it became from the auto-upgrade tool.